### DISCUSSION TITLE
Grafana question - is it possible to have something what "My basket" in PNP did?

### DISCUSSION THREAD

**Discussion by PepeOnAChair** [sni/thruk#1195] [Q&A]
Hello again,

I know this question is not much related to Thruk, but to Grafana, but I guess it would be hard to explain what "My basket" is in PNP graphs to pure Grafana people.

In PNP, one can select graphs by green plus icon in the upper right corner of each graph, and add the graph to "My basket". Than, after clicking to "Show basket" user can analyze graphs from different hosts/services/datasources and alling their graphs with same time interval. So it is easilly visible, if some graph line drop correlates with other, for example. (Actually I beleive you know what I`m talking about :) Is it somehow possible to have such feature in Grafana, please?

Thank you
Pep.

**Comment by sni** [comment]
Moved to discussions... This is not an issue with Thruk. Personally i don't think anyone from the
grafana team will read this here so i'd suggest to open an issue there and try to explain what you
would like to see.
